ProcessLib/BoundaryCondition/BoundaryCondition.h

```diff
@@ -11,125 +11,127 @@
 
 #include "NumLib/NumericsConfig.h"
 
-namespace GeoLib
-{
+namespace GeoLib {
 class GeoObject;
 }
 
-namespace MeshLib
-{
+namespace MeshLib {
 class Element;
 class Mesh;
-}
+} // namespace MeshLib
 
-namespace MeshGeoToolsLib
-{
+namespace MeshGeoToolsLib {
 class SearchLength;
 class MeshNodeSearcher;
 class BoundaryElementsSearcher;
-}
+} // namespace MeshGeoToolsLib
 
-namespace NumLib
-{
+namespace NumLib {
 class LocalToGlobalIndexMap;
-template <typename>
-struct IndexValueVector;
-}
+template <typename> struct IndexValueVector;
+} // namespace NumLib
 
-namespace ProcessLib
-{
+namespace ProcessLib {
 struct BoundaryConditionConfig;
 struct ParameterBase;
 
-class BoundaryCondition
-{
+class BoundaryCondition {
 public:
-    //! Applies natural BCs (i.e. non-Dirichlet BCs) to the stiffness matrix
-    //! \c K and the vector \c b.
-    virtual void applyNaturalBC(const double /*t*/, GlobalVector const& /*x*/,
-                                GlobalMatrix& /*K*/, GlobalVector& /*b*/)
-    {
-        // By default it is assumed that the BC is not a natural BC. Therefore
-        // there is nothing to do here.
-    }
-
-    //! Writes the values of essential BCs to \c bc_values.
-    virtual void getEssentialBCValues(
-        const double /*t*/,
-        NumLib::IndexValueVector<GlobalIndexType>& /*bc_values*/) const
-    {
-        // By default it is assumed that the BC is not an essential BC.
-        // Therefore there is nothing to do here.
-    }
-
-    virtual ~BoundaryCondition() = default;
+  //! Applies natural BCs (i.e. non-Dirichlet BCs) to the stiffness matrix
```

Comment:
Please revert the formatting to the original. There is .clang-format file in the source code tree which must be used with clang-format. Otherwise some other default formatting is used. There must be an option in qtcreator to set the formatting style to "file".

Reply:
clang-format has been re-run.

Reply:
It looks that  the clang-format in this file has been changed.